addons/bus/tests/test_health.py

```diff
@@ -0,0 +1,9 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import HttpCase
```

Comment:
did you mean to import this testfile somewhere?

Reply:
Whoops! Lost the import when I rebased from 13 to master. Fixed.